### PR TITLE
Update docs to follow up config format changes.

### PR DESCRIPTION
**Pull request by wi1dcard**

### CODE REVIEW

**Comment by wi1dcard** [comment]
I've just submitted the endpoint changes in this PR, please review and merge it once the changes have been announced.

File: .maintain/config/mainnet.sample.yml [hunk 1]
```diff
@@ -42,11 +42,11 @@ ethereum:
   authority:
     # private key to sign ecdsa messages
     # the signature will be submitted to Darwinia by darwinia.relayer
-    private_key: 0x... 
+    private_key: 0x...
 
 darwinia:
   # darwinia node websocket json-rpc endpoint
-  rpc: wss://cc1.darwinia.network
+  rpc: wss://mainnet.darwinia.network
```
Comment (HackFisher) [inline .maintain/config/mainnet.sample.yml:49]: Check

File: README.md [hunk 2]
```diff
@@ -42,66 +42,72 @@ cargo build --release
...
-#### `node`
+#### `darwinia.rpc`
 
 The WebSocket endpoint of Darwinia official node or any archive node (`darwinia --pruning=archive`), supports `ws://` and `wss://`. For example:
 
-```toml
-node = "wss://cc1.darwinia.network"
+```yaml
+darwinia:
+  rpc: wss://mainnet.darwinia.network
```
Comment (HackFisher) [inline README.md:60]: check

**Comment by HackFisher** [comment]
Does crab config need to update accordingly?

`crab-rpc.darwinia.network`

https://github.com/darwinia-network/bridger/blob/20a5a3acdbc71845be8a8b425ac0d55c865c4a9b/.maintain/config/ropsten_crab.sample.yml#L50

**Review by HackFisher — APPROVED** [review]

**Pull request merged**